### ISSUE TITLE
Prices for no-symbol commodity

### ISSUE THREAD

**Issue by ceracera** [simonmichael/hledger#1741]
I use a no-symbol commodity for most postings since it's easier to type and takes less space in reports. It's easier but I have a problem with prices.

Let's say my default currency is Serbian dinar, RSD. When I try to set a market price I have a problem.

Eg, euro to dinars. One euro is 117,58 RSD. This way it's ok since I use symbol for euros:
`P 2021/10/25 EUR 117,58`

Now dinar to euros. I want to show that one dinar is 0,0085 euros. This way it's not possible:
`P 2021/10/25 0,0085 EUR`

Can it be done somehow?

**Comment by simonmichael** [comment]
Yes I'd like if this could be made to work somehow. For now, you can use a [default commodity](https://hledger.org/hledger.html#default-commodity) directive to give all of those a commodity symbol.

**Comment by Xitian9** [comment]
Would it be sufficient to extend the syntax to allow the following, or will the used of double quotes to indicate an empty commodity cause problems?

```
P 2021/10/25 "" 0,0085 EUR
```

**Comment by simonmichael** [comment]
I don't remember any reason why this wouldn't work - sounds good to me. 

It seems `""` is currently not accepted as a commodity symbol anywhere. Perhaps it should be, just for consistency (it would be equivalent to writing no symbol).

Interestingly `" "`, `"  "`, `"   "` etc. are accepted, and distinct. We should probably strip outer whitespace from double quoted commodity symbols.
